crypto/keys/pem_key_provider.go

```diff
@@ -0,0 +1,35 @@
+package keys
+
+import (
+	"context"
+	"crypto"
+	"fmt"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/trillian"
+)
+
+// PEMKeyProvider loads PEM-encoded private keys.
+// It only supports trees whose PrivateKey field is a trillian.PEMKeyFile.
+// It implements keys.Provider.
+// TODO(robpercival): Should this cache loaded private keys? The SequenceManager will request a signer for each batch of leaves it sequences.
```

Comment:
Suggestion: If that's only an issue for the sequencer, perhaps we should cache it there instead.

(No need to do any changes to the PR because of this comment.)